integration-test/run-separate/dlcManager.test.ts

```diff
@@ -173,29 +174,9 @@ describe('dlc-manager', () => {
     remotePartyManager.finalize()
   })
 
-  test('20-mutual-closing-both-send', async () => {
+  test('18-closed-by-local', async () => {
     const contractId = await commonTests()
-    await bothReceiveMutualClose(contractId)
-  })
-
-  test('21-mutual-closing-local-send', async () => {
-    const contractId = await commonTests()
-    await onlyLocalSendsMutualClosing(contractId)
-  })
-
-  test('22-unilateral-close-by-local', async () => {
-    const contractId = await commonTests()
-    await unilateralCloseFromLocal(contractId)
-  })
-
-  test('23-unilateral-close-by-remote', async () => {
-    const contractId = await commonTests()
-    await unilateralCloseFromRemote(contractId)
-  })
-
-  test('24-unilateral-close-by-local-after-timeout', async () => {
-    const contractId = await commonTests()
-    await unilateralCloseFromLocalAfterProposeTimeout(contractId)
+    await closedByLocal(contractId)
```

Comment:
`expect(closedByLocal(contractId)).to.resolves()` ?

Reply:
This is a test function, not sure what the expect would be checking?